back/app/models/concerns/post.rb

```diff
@@ -54,10 +54,13 @@ module Post
     scope :published, -> { where publication_status: 'published' }
 
     scope :order_new, ->(direction = :desc) { order(published_at: direction) }
-    scope :order_random, lambda {
-      modulus = RandomOrderingService.new.modulus_of_the_day
+    scope :order_random, lambda { |user|
+      modulus = RandomOrderingService.new.modulus_of_the_day(user)
       order(Arel.sql("(extract(epoch from #{table_name}.created_at) * 100)::bigint % #{modulus}, #{table_name}.id"))
```

Comment:
As you suggested I do think that using the `id` instead of `created_at` would be better here, and would fix the issue of imported ideas not being so random

Reply:
Alright, looking into it.